docgen/src/guides/v3-migration.md

```diff
@@ -162,6 +162,89 @@ Here are the elements you need to migrate:
 </div>
 ```
 
+### CurrentRefinements (formerly CurrentRefinedValues)
+
+#### Options
+
+| Before                 | After                             |
+| ---------------------- | --------------------------------- |
+| `attributes`           | `includedAttributes`              |
+| `onlyListedAttributes` | use `includedAttributes`          |
+| `clearAll`             | use the `clearRefinements` widget |
+| `clearsQuery`          | `includesQuery`                   |
+
```

Comment:
Add `excludedAttributes` as a new option

Reply:
possible solution for query @samouss: default value of `excludedAttributes` can be `["query"]`, and then we don't need `includeQuery` anymore.